libnavui-maneuver/src/test/java/com/mapbox/navigation/ui/maneuver/TurnIconHelperTest.kt

```diff
@@ -35,10 +35,10 @@ class TurnIconHelperTest {
         val mockType: String? = null
         val mockDegrees: Float? = null
         val mockModifier: String? = null
-        val mockDrivingSide: String? = null
+        val mocDRIVINGSide: String? = null
```

Comment:
typo?
cc @LukasPaczos

Reply:
ah, `find and replace` issue, I'll revert